### PR TITLE
Adding Threshhold templating and monitor level default definitions

### PR DESCRIPTION
**Pull request by ejether**
added threshold templating definitions, added threshold template block in rodd, dry up the definition replacement

### CODE REVIEW

**Comment by ejether** [comment]
closes: #1

File: pentagon_datadog/rodd.py [hunk 1]
```diff
@@ -122,22 +122,32 @@ def definitions(self):
...
                 for index, item in enumerate(data[key]):
-                    for var, value in self.definitions.iteritems():
-                        data[key][index] = data[key][index].replace("${%s}" % str(var), value)
-
-            if type(data[key]) == str or type(data) == unicode:
-                for var, value in self.definitions.iteritems():
-                    logging.debug("Replacing Definition: {}:{}".format(var, value))
-                    data[key] = data[key].replace("${%s}" % str(var), value)
+                    data[key][index] = _replace_definition(data[key][index], _definitions)
+            elif key == 'thresholds':
```
Comment (ejether) [inline pentagon_datadog/rodd.py]: @sudermanjr  I'm so sorry.

Comment (sudermanjr) [reply pentagon_datadog/rodd.py]: You are forgiven.

File: docs/monitors.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
+# List of Monitor Families and Monitors;
```
Comment (sudermanjr) [inline docs/monitors.md]: This is super awesome.  Let's make sure to keep it up to date.  I leave it to the CODEOWNERS to not approve a PR that dosen't update this.

Comment (mjhuber) [reply docs/monitors.md]: idea for future enhancement: auto document each monitor

**Comment by mjhuber** [comment]
great contribution - thanks for submitting this!

File: docs/monitors.md (hunk 2 above)
Comment (ejether) [reply docs/monitors.md]: Sounds like a cool idea.

Comment (ejether) [reply docs/monitors.md]: https://github.com/reactiveops/rodd/issues/47

**Comment by crmejia** [comment]
LGTM

**Pull request merged**